turbopack/crates/turbo-tasks-macros-tests/tests/value_trait/pass_trait_items.rs

```diff
@@ -13,6 +13,13 @@ trait MyTrait {
 
     #[turbo_tasks::function]
     fn item3(&self) -> Vc<bool>;
+
+    fn item4(self: Vc<Self>) -> bool
```

Comment:
Is it still possible to override this method, or is that just a new syntax for an extension method?